content/en/docs/components/trainer/user-guides/pytorch.md

```diff
@@ -4,4 +4,293 @@ description = "How to develop PyTorch models with Kubeflow Trainer"
 weight = 10
 +++
 
-This doc is in progress...
+This page describes how to use `PyTorchJob` for training a machine learning model with [PyTorch](https://pytorch.org/).
+
+The `PyTorchJob` is a Kubernetes
+[custom resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+to run PyTorch training jobs on Kubernetes. The Kubeflow implementation of
+the `PyTorchJob` is in the [`trainer`](https://github.com/kubeflow/trainer).
+
+## Creating a PyTorch Training Job
+
+### Define the Training Function
```

Comment:
We need to figure out how to avoid duplications with this Getting Started section: https://www.kubeflow.org/docs/components/trainer/getting-started/#getting-started-with-pytorch